src/docs/output-targets/custom-elements.md

```diff
@@ -115,19 +130,42 @@ Make sure to copy the assets over to a public directory in your app. This config
 bundling, and where your assets can be loaded from. How the files are copied to the production build directory depends on the bundler or tooling.
 The configs below provide examples of how to do this automatically with popular bundlers.
 
+## Consuming Custom Elements
```

Comment:
I'd consider putting this above the "Making Assets Available". I have a feeling this will get more views than that section (but I could be wrong 🤷)